src/App.vue

```diff
@@ -1,22 +1,30 @@
 <template>
 	<v-app>
-		<app-bar logo="header-logo-white.png" :links="headerLinks"></app-bar>
+		<v-row>
+			<v-col cols="10" class="offset-1">
+				<div style="position:relative;">
+					<app-bar logo="header-logo-white.png" :links="headerLinks"></app-bar>
+				</div>
+			</v-col>
+		</v-row>
 		<v-main>
-			<section-landing title="Provided Page Title" background-image="https://i.imgur.com/bX2Uk7C.jpg">
+			<section-landing title="SEARCH AND YOU SHALL FIND. DON’T MISS MAKTUB" background-image="/maktub-banner.png">
```

Comment:
Please move all the images to the `/assets/image` directory and then use them like this: 

`:background-image="require(`@/assets/image/maktub-banner.png`)"`

That way we can easily swap out images by making them variables.

Reply:
Ah sure, doing that quickly.

Reply:
@angelomelonas Completed. I implemented the same concept as with `AppBar` and `FooterSimple`. You pass through the name of the image, which is used as a prop in the child component. So in the child component it is used like `:src="require(\`@/assets/image/${image}\`)"`